Sources/Segment/Startup.swift

```diff
@@ -40,20 +40,24 @@ extension Analytics: Subscriber {
         // add context plugin as well as it's platform specific internally.
         // this must come first.
         plugins.append(Context())
+        
+        plugins += VendorSystem.current.requiredPlugins
 
         // setup lifecycle if desired
         if configuration.values.trackApplicationLifecycleEvents {
             #if os(iOS) || os(tvOS)
-            plugins += [iOSLifecycleMonitor(), iOSLifecycleEvents(), DeviceToken()]
+            plugins.append(iOSLifecycleEvents())
```

Comment:
would it be cleaner to provide the lifecycleEvents plugin via the vendor system as well?

Reply:
I think that would add clutter in the long term if/when we have more things that fall into that bucket.